tests/unit/containers/email-verification/EmailVerification.spec.jsx

```diff
@@ -0,0 +1,58 @@
+import { describe, it, expect, vi } from 'vitest'
```

Comment:
You can remove describe, it and expect from import
```suggestion
import { vi } from 'vitest'
```